srcpkgs/boot-66serv/files/switch-initutils-66

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Don't need the empty line, I think?

Reply:
Not needed,  just aesthetics :) I can remove it if it is a problem...